sdk/inc/azure/core/internal/az_hfsm_pipeline_internal.h

```diff
@@ -0,0 +1,180 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file az_hfsm_pipeline.h
```

Comment:
you can leave this blank to prevent wrong filenames (like here 😁)
```suggestion
 * @file
```